hexrd/ui/resources/ui/import_data_panel.ui

```diff
@@ -7,26 +7,13 @@
     <x>0</x>
     <y>0</y>
     <width>491</width>
-    <height>711</height>
+    <height>472</height>
```

Comment:
this is messy, sorry. The size is irrelevant here, yes?

Reply:
Yeah, I don't think it matters.